PokerChallenge/PokerChallenge(2)/PokerHand.cs

```diff
@@ -0,0 +1,53 @@
+﻿using PokerChallenge2.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PokerChallenge2
+{
+    public class PokerHand
+    {
+        public List<Card> Cards { get; set; }
+        public HandType Type { get { return EvaluateHand(); } }
```

Comment:
This looks like a function with extra steps. I would prefer to see the hand type not be such a complex function. A caller might access this and have no idea its sorting cards (which is changing state) or doing all this work. It would better if this returned by default a "not evaluated" value which allows the caller to get better insight. Then you could expose the Evaluate as public and give the caller more control.